test/common/runtime/utility.h

```diff
@@ -8,14 +8,33 @@ namespace Runtime {
 
 class RuntimeFeaturesPeer {
 public:
-  static bool addFeature(const std::string& feature) {
-    return const_cast<Runtime::RuntimeFeatures*>(&Runtime::RuntimeFeaturesDefaults::get())
-        ->enabled_features_.insert(feature)
-        .second;
+  static bool addFeature(const std::string& feature, bool disable) {
```

Comment:
Maybe we call these enableFeature and disableFeature, and always try (but OK to fail) to remove from the opposite list, then add it to the appropriate list?  
Then we move the bool to outside, but at the call site it's really clear what operation we're performing.

Reply:
done, much nicer! i felt compelled then to add a test to make sure that no-op runtime overrides would work nicely for both enabling/disabling